src/k8s-configuration/setup.py

```diff
@@ -16,7 +16,7 @@
 
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
 
-VERSION = "1.7.0"
+VERSION = "1.8.0"
```

Comment:
breaking change in new release should increase the major version number to be `2.0.0`. For cli extension versioning shcema, please refer to [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)